packages/remote/src/electron-browser/remote-frontend-module.ts

```diff
@@ -65,4 +70,19 @@ export default new ContainerModule((bind, _, __, rebind) => {
     bind(RemotePortForwardingProvider).toDynamicValue(ctx =>
         ServiceConnectionProvider.createLocalProxy<RemotePortForwardingProvider>(ctx.container, RemoteRemotePortForwardingProviderPath)).inSingletonScope();
 
+    bind(LocalRemoteFileSytemServer).toDynamicValue(ctx =>
+        isRemoteConnection() ?
+            ServiceConnectionProvider.createLocalProxy(ctx.container, remoteFileSystemPath, new RemoteFileSystemProxyFactory()) :
+            ctx.container.get(RemoteFileSystemServer));
+    bind(LocalEnvVariablesServer).toDynamicValue(ctx =>
+        isRemoteConnection() ?
+            ServiceConnectionProvider.createLocalProxy<EnvVariablesServer>(ctx.container, envVariablesPath) :
+            ctx.container.get(EnvVariablesServer));
+    bind(LocalRemoteFileSystemProvider).toSelf().inSingletonScope();
+    rebind(UserStorageContribution).to(RemoteUserStorageContribution);
+
 });
+
+function isRemoteConnection(): boolean {
+    return !!new URLSearchParams(window.location.search).get('localPort');
+}
```

Comment:
```suggestion
export function isRemoteConnection(): boolean {
    return !!new URLSearchParams(window.location.search).get('localPort');
}
```

Should we maybe export this, to make it easier for adoptors to rebind?
Also is there a way to fetch the 'localPort' string from somewhere and not hard code it here?

Reply:
is changed it to use the isRemote flag from the browser.ts. Also unified all calls to get the "port" and "localPort" url search query params